ide/ideutils.ml

```diff
@@ -458,15 +458,6 @@ let browse prerr url =
   in
   run_command (fun _ -> ()) finally com
 
-let doc_url () =
```

Comment:
Removing this makes the documentation no longer work when disconnected from the internet, even if it's installed locally, right?  (I don't object, because I don't use CoqIDE, I just want to make sure this is deliberate.)

Reply:
I personally find it fairly nonsensical to have a link to a website inside a GUI. At best this helps total beginners who have no idea on where to find the doc, but apart from that, I really don't see why you wouldn't just open your favourite source of documentation by standard means.

Reply:
It makes (some) sense to have a link that points to the doc *for the version you are using*.
Or at least, this is the reason why I did not zap entirely this feature.

@JasonGross yes it is deliberate.